satpy/tests/reader_tests/test_aws1_mwr_l1c.py

```diff
@@ -0,0 +1,99 @@
+file_pattern = "W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1C-RAD_C_OHB__{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc"  # noqa
+
+
+geo_dims = ["n_scans", "n_fovs"]
+geo_size = 10*145
+fake_lon_data, fake_lat_data = make_fake_mwr_l1c_lonlats(geo_size, geo_dims)
+fake_sun_azi_data = make_fake_angles(geo_size, geo_dims, shape=(10, 145))
+fake_sun_zen_data = make_fake_angles(geo_size, geo_dims, shape=(10, 145))
+fake_sat_azi_data = make_fake_angles(geo_size, geo_dims, shape=(10, 145))
+fake_sat_zen_data = make_fake_angles(geo_size, geo_dims, shape=(10, 145))
+
+
+def test_get_channel_data(aws_mwr_l1c_handler, fake_mwr_data_array):
+    """Test retrieving the channel data."""
+    did = dict(name="1")
+    dataset_info = dict(file_key="data/calibration/aws_toa_brightness_temperature")
+    expected = fake_mwr_data_array.isel(n_channels=0)
+    # mask no_data value
+    expected = expected.where(expected != -2147483648)
+    # mask outside the valid range
+    expected = expected.where(expected <= 700000)
+    expected = expected.where(expected >= 0)
+    # "calibrate"
+    expected = expected * 0.001
+    res = aws_mwr_l1c_handler.get_dataset(did, dataset_info)
+    np.testing.assert_allclose(res, expected)
+    assert "x" in res.dims
+    assert "y" in res.dims
+    assert res.dims == ("y", "x")
+    assert "n_channels" not in res.coords
+    assert res.attrs["sensor"] == "MWR"
+    assert res.attrs["platform_name"] == "AWS1"
+
+
+@pytest.mark.parametrize(("id_name", "file_key", "fake_array"),
+                         [("longitude", "data/navigation/aws_lon", fake_lon_data * 1e-4),
+                          ("latitude", "data/navigation/aws_lat", fake_lat_data),
+                          ])
+def test_get_navigation_data(aws_mwr_l1c_handler, id_name, file_key, fake_array):
+    """Test retrieving the geolocation (lon, lat) data."""
+    did = dict(name=id_name)
+    dataset_info = dict(file_key=file_key, standard_name=id_name)
+    res = aws_mwr_l1c_handler.get_dataset(did, dataset_info)
+    if id_name == "longitude":
+        fake_array = fake_array.where(fake_array <= 180, fake_array - 360)
+
+    np.testing.assert_allclose(res, fake_array)
+    assert "x" in res.dims
+    assert "y" in res.dims
+    assert res.dims == ("y", "x")
+    assert "standard_name" in res.attrs
+    if id_name == "longitude":
+        assert res.max() <= 180
+
+
+@pytest.mark.parametrize(("id_name", "file_key", "fake_array"),
+                         [("solar_azimuth", "data/navigation/aws_solar_azimuth_angle", fake_sun_azi_data),
+                          ("solar_zenith", "data/navigation/aws_solar_zenith_angle", fake_sun_zen_data),
+                          ("satellite_azimuth", "data/navigation/aws_satellite_azimuth_angle", fake_sat_azi_data),
+                          ("satellite_zenith", "data/navigation/aws_satellite_zenith_angle", fake_sat_zen_data)])
```

Comment:
My consistency suggestion in the YAML and reader parts add the `_angle` ending
```suggestion
@pytest.mark.parametrize(("id_name", "file_key", "fake_array"),
                         [("solar_azimuth_angle", "data/navigation/aws_solar_azimuth_angle", fake_sun_azi_data),
                          ("solar_zenith_angle", "data/navigation/aws_solar_zenith_angle", fake_sun_zen_data),
                          ("satellite_azimuth_angle", "data/navigation/aws_satellite_azimuth_angle", fake_sat_azi_data),
                          ("satellite_zenith_angle", "data/navigation/aws_satellite_zenith_angle", fake_sat_zen_data)])
```